app/presenters/track.rb

```diff
@@ -47,12 +47,12 @@ def topic_content(topic_key)
       end
 
       def contributing_instructions(topic_key)
-        template = File.read("./x/docs/md/track/BETTER.md")
+        template = File.read(shared_template_filepath("BETTER"))
         populate_template_with_trackler_data(template).gsub('TOPIC', topic_key.to_s.upcase)
       end
 
       def fallback_topic_content(topic_key)
```

Comment:
I'm a little torn. Do you think it's important that whatever's calling these methods knows how `topic_key` will be used for?

Here's the place where we're calling these methods now:

```
def topic_content(topic_key)
  if trackler_topic_content(topic_key).present?
    [trackler_topic_content(topic_key).strip, contributing_instructions(topic_key)].join("\n")
  else
    fallback_topic_content(topic_key)
  end
end
```

I think it's nice that `topic_content` doesn't need to know the implementation of `contributing_instructions` or `fallback_topic_content`. Instead, it passes along the `topic_key` and it gets back the relevant content.